client/src/Platform.jsx

```diff
@@ -16,7 +16,7 @@ import './styles/global.css';
  */
 const Platform = () => {
   /* Configure global Axios defaults */
-  axios.defaults.baseURL = '/api/v1';
+  axios.defaults.baseURL = `${import.meta.env.VITE_API_BASE_URL}/api/v1`;
```

Comment:
This should be conditionally included only during development. If left as an absolute URL, the client JS built at Docker build time will always point to LibreCommons, even if it is a Campus Commons instance.

Reply:
Fixed